lua/telescope/builtin/files.lua

```diff
@@ -145,10 +145,25 @@ files.grep_string = function(opts)
   }):find()
 end
 
--- TODO: Maybe just change this to `find`.
--- TODO: Support `find` and maybe let people do other stuff with it as well.
-files.find_files = function(opts)
-  local find_command = opts.find_command
+local function get_find_command(opts)
```

Comment:
either move this local function inside `builtin.find_files` or do something like this

```lua
local find_command = (function()
  if opts.find_command then
    return opts.find_command
  elseif 1 == vim.fn.executable "fd" then
    return { "fd", "--type", "f" }
  elseif 1 == vim.fn.executable "fdfind" then
    return { "fdfind", "--type", "f" }
  elseif 1 == vim.fn.executable "rg" then
    return { "rg", "--files" }
  elseif 1 == vim.fn.executable "find" and vim.fn.has "win32" == 0 then
    return { "find", ".", "-type", "f" }
  elseif 1 == vim.fn.executable "where" then
    return { "where", "/r", ".", "*" }
  end
end)()
```

There is no need to have a local function that is only used once